src/Collector.cpp

```diff
@@ -363,3 +366,36 @@ void CollectorTime::setValue(int i, const Token& t) {
   }
 
 }
+
+void CollectorDeparsed::setValue(int i, const Token& t) {
+  switch(t.type()) {
+  case TOKEN_STRING: {
+    boost::container::string buffer;
+    SourceIterators string = t.getString(&buffer);
+    std::string std_string(string.first, string.second);
+    RObject ans = R_NilValue;
+
+    ParseStatus status;
+
+    CharacterVector c(std_string);
+    ExpressionVector r = R_ParseVector(c, -1, &status, R_NilValue);
+    if (status == PARSE_OK) {
+      for(R_len_t i = 0; i < Rf_length(r); i++) {
+        ans = Rf_eval(VECTOR_ELT(r, i), R_GlobalEnv);
```

Comment:
I don't think evaluating is that bad, you have to opt in for this so presumably you trust the input in that column.

We could just return the unevaluated expressions, but I think it would just make people perform an extra step the vast majority of the time. 

Although `x$col <- lapply(x$col, eval)` is not that bad and would allow people to inspect the expressions before evaluating them.

We could maybe make this an argument as well...